sdk/core/azure-core/src/main/java/com/azure/core/credential/SimpleTokenCache.java

```diff
@@ -18,6 +18,7 @@
  */
 public class SimpleTokenCache {
     private static final Duration REFRESH_TIMEOUT = Duration.ofSeconds(30);
```

Comment:
I know this isn't part of this PR, but I feel that `REFRESH_TIMEOUT` doesn't really describe what this value is used for, and it makes the code a bit harder to understand. Would `REFRESH_DELAY` be more appropriate?

Reply:
Updated.

Reply:
Also added some comments to increase readability